docs/UsersGuide/source/BuildingRunningTesting/RunSRW.rst

```diff
@@ -0,0 +1,1206 @@
+   | Cheyenne     | /glade/work/epicufsrt/contrib/UFS_SRW_data/develop/input_model_data/         |
+   +--------------+------------------------------------------------------------------------------+
+   | Gaea         | /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/      |
+   +--------------+------------------------------------------------------------------------------+
+   | Hera         | /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/input_model_data/      |
+   +--------------+------------------------------------------------------------------------------+
+   | Jet          | /mnt/lfs4/HFIP/hfv3gfs/role.epic/UFS_SRW_data/develop/input_model_data/      |
+   +--------------+------------------------------------------------------------------------------+
+   | NOAA Cloud   | /contrib/EPIC/UFS_SRW_data/develop/input_model_data/                         |
+   +--------------+------------------------------------------------------------------------------+
+   | Orion        | /work/noaa/epic-ps/role-epic-ps/UFS_SRW_data/develop/input_model_data/       |
+   +--------------+------------------------------------------------------------------------------+
+   | WCOSS2       | /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/input_model_data/                |
+   +--------------+------------------------------------------------------------------------------+ 
+    
+For Level 2-4 systems, the data must be added to the user's system. Detailed instructions on how to add the data can be found in :numref:`Section %s: Downloading and Staging Input Data <DownloadingStagingInput>`. Sections :numref:`%s <Input>` and :numref:`%s <OutputFiles>` contain useful background information on the input and output files used in the SRW App. 
+
+.. _GridSpecificConfig:
+
+Grid Configuration
+=======================
+
+The SRW App officially supports the four predefined grids shown in :numref:`Table %s <PredefinedGrids>`. The out-of-the-box SRW App case uses the ``RRFS_CONUS_25km`` predefined grid option. More information on the predefined and user-generated grid options can be found in :numref:`Section %s: Limited Area Model (LAM) Grids <LAMGrids>`. Users who plan to utilize one of the four predefined domain (grid) options may continue to the next step (:numref:`Step %s: Generate the Forecast Experiment <GenerateForecast>`). Users who plan to create a new custom predefined grid should refer to the instructions in :numref:`Section %s: Creating User-Generated Grids <UserDefinedGrid>`. At a minimum, these users will need to add the new grid name to the ``valid_param_vals.yaml`` file and add the corresponding grid-specific parameters in the ``predef_grid_params.yaml`` file.
+
+.. _PredefinedGrids:
+
+.. table::  Predefined Grids Supported in the SRW App
+
+   +----------------------+-------------------+--------------------------------+
+   | **Grid Name**        | **Grid Type**     | **Quilting (write component)** |
```

Comment:
Honestly, no idea! Christina thought that some of the grids might be deprecated (at least from the perspective of use--they're still listed in the valid params file). I think this would be a good thing to bring up at the CM and/or SRW release meetings.